### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by tapabratadey**

### CODE REVIEW

File: src/backend/config/db.ts [hunk 1]
```diff
@@ -7,7 +7,8 @@ import { Like } from '../entities/Like';
 import { Category } from '../entities/Category';
 
 import chalk from 'chalk';
-
+/**
```
Comment (sheikhrachel) [inline src/backend/config/db.ts:10]: remove commented code

File: src/backend/controller/comment/comment.ts [hunk 2]
```diff
@@ -1,25 +1,28 @@
...
 import { Request } from '@backend/middleware/verify_request';
-import {
-  DatabaseError
-} from 'helpers/ErrorHandling/errors';
+import { DatabaseError } from 'helpers/ErrorHandling/errors';
 
 const commentController = {
   getAllById: async (req: Request) => {
-    return commentService.getAllById(req.body, req.query);
+    console.log('in the controller');
```
Comment (sheikhrachel) [inline src/backend/controller/comment/comment.ts:8]: remove log

File: src/backend/controller/project/ProjectController.ts [hunk 3]
```diff
@@ -50,8 +55,10 @@ const ProjectController = {
   // READ - 1
   findById: async (req) => {
     try {
+      const { id } = req.query;
+      console.log(chalk.green('@line 59: ', id));
```
Comment (sheikhrachel) [inline src/backend/controller/project/ProjectController.ts:59]: remove log

File: src/backend/services/db/comment/commentService.ts [hunk 4]
```diff
@@ -4,45 +4,48 @@ import { Comment } from '@backend/entities/Comment';
...
-            .from(Comment, 'comment')
-            .innerJoin(User, 'user', 'comment.userId = user.id')
-            .where('"projectId" = :id', { id })
-            .getRawMany();
-        return comments;
-    }
-    catch {
-        throw new DatabaseError(dbError);
+      const db = await connection();
+      console.log(chalk.green(getConnection().isConnected));
```
Comment (sheikhrachel) [inline src/backend/services/db/comment/commentService.ts:18]: remove log

File: src/backend/services/db/comment/commentService.ts [hunk 5]
```diff
@@ -4,45 +4,48 @@ import { Comment } from '@backend/entities/Comment';
...
+import chalk from 'chalk';
 
 const commentService = {
   //Add a like to a project by a user
-    getAllById: async (body, query) => {
+  getAllById: async (query) => {
     const id = query.id;
-    const db = await connection();
-
+    console.log(chalk.green(id));
```
Comment (sheikhrachel) [inline src/backend/services/db/comment/commentService.ts:15]: remove log

File: src/backend/services/db/comment/commentService.ts [hunk 6]
```diff
@@ -4,45 +4,48 @@ import { Comment } from '@backend/entities/Comment';
...
+      console.log(chalk.green(getConnection().isConnected));
+      console.log(chalk.green('================='));
+      const comments = await db
+        .createQueryBuilder()
+        .select('*')
+        .from('comment', 'comment')
+        .innerJoin('user', 'user', 'comment.userId = user.id')
+        .where('"projectId" = :id', { id })
+        .getRawMany();
+      console.log(comments);
```
Comment (sheikhrachel) [inline src/backend/services/db/comment/commentService.ts:27]: remove log

File: src/backend/services/db/comment/commentService.ts [hunk 7]
```diff
@@ -4,45 +4,48 @@ import { Comment } from '@backend/entities/Comment';
...
+      return comments;
+    } catch (err) {
+      console.log(chalk.red('================='), err);
```
Comment (sheikhrachel) [inline src/backend/services/db/comment/commentService.ts:30]: remove log

File: src/backend/services/db/project/ProjectService.ts [hunk 8]
```diff
@@ -73,12 +73,19 @@ const ProjectService = {
...
+        .addSelect('project.pictures', 'projectImageUrl')
+        .addSelect('user.firstName', 'firstName')
+        .addSelect('user.lastName', 'lastName')
         .from('project', 'project')
+        .innerJoin('user', 'user', 'project.userId = user.id')
         .where(
           `project.user = (SELECT id FROM public.user WHERE uid = '${uid}')`
         )
         .getRawMany();
+      console.log(userData);
```
Comment (sheikhrachel) [inline src/backend/services/db/project/ProjectService.ts:88]: remove log

File: src/backend/services/db/project/ProjectService.ts [hunk 9]
```diff
@@ -114,21 +134,23 @@ const ProjectService = {
...
+      console.log(chalk.green('@line 128: ', id));
       const foundProject = await db
         .createQueryBuilder()
         .select('*')
         .from('project', 'project')
-        .where(`project.id = ${findByIdParams.id}`)
+        .where(`project.id = :id`, { id })
         .getRawOne();
-
+      console.log('================', foundProject);
```
Comment (sheikhrachel) [inline src/backend/services/db/project/ProjectService.ts:153]: remove log

File: src/backend/services/db/project/ProjectService.ts [hunk 10]
```diff
@@ -114,21 +134,23 @@ const ProjectService = {
...
-  findById: async (findByIdParams) => {
+  findById: async (id) => {
     try {
       const db = await connection();
 
       if (db === undefined || db === null) {
         throw new DatabaseError(dbError);
       }
-
+      console.log(chalk.green(getConnection().isConnected));
```
Comment (sheikhrachel) [inline src/backend/services/db/project/ProjectService.ts:144]: remove log

File: src/backend/services/db/project/ProjectService.ts [hunk 11]
```diff
@@ -138,16 +160,17 @@ const ProjectService = {
       }
       return [foundProject, StatusCodes.OK];
     } catch (err) {
+      console.log(chalk.red('================= @Line 144: '), err);
```
Comment (sheikhrachel) [inline src/backend/services/db/project/ProjectService.ts:163]: remove log

File: src/backend/services/db/project/ProjectService.ts [hunk 12]
```diff
@@ -320,16 +343,20 @@ const ProjectService = {
...
+    .addSelect('project.title', 'title')
+    .addSelect('project.description', 'description')
+    .addSelect('project.category', 'category')
+    .addSelect('user.firstName', 'firstName')
+    .addSelect('user.lastName', 'lastName')
+    .from('project', 'project')
+    .innerJoin(User, 'user', 'project.userId = user.id')
+    .orderBy(`project.${column}`, 'DESC')
+    .getRawMany();
+  //console.log(projectData);
```
Comment (sheikhrachel) [inline src/backend/services/db/project/ProjectService.ts:358]: remove log

File: src/backend/config/db.ts (hunk 1 above)
Comment (tapabratadey) [reply src/backend/config/db.ts]: removed

File: src/backend/controller/project/ProjectController.ts (hunk 3 above)
Comment (tapabratadey) [reply src/backend/controller/project/ProjectController.ts]: removed

File: src/backend/services/db/comment/commentService.ts (hunk 4 above)
Comment (tapabratadey) [reply src/backend/services/db/comment/commentService.ts]: removed

File: src/backend/services/db/project/ProjectService.ts (hunk 8 above)
Comment (tapabratadey) [reply src/backend/services/db/project/ProjectService.ts]: removed

File: src/backend/services/db/project/ProjectService.ts (hunk 10 above)
Comment (tapabratadey) [reply src/backend/services/db/project/ProjectService.ts]: removed

File: src/backend/services/db/project/ProjectService.ts (hunk 12 above)
Comment (tapabratadey) [reply src/backend/services/db/project/ProjectService.ts]: removed

File: src/backend/services/db/project/ProjectService.ts (hunk 11 above)
Comment (tapabratadey) [reply src/backend/services/db/project/ProjectService.ts]: removed

File: src/backend/services/db/project/ProjectService.ts (hunk 9 above)
Comment (tapabratadey) [reply src/backend/services/db/project/ProjectService.ts]: removed

File: src/backend/services/db/comment/commentService.ts (hunk 5 above)
Comment (tapabratadey) [reply src/backend/services/db/comment/commentService.ts]: removed

File: src/backend/controller/comment/comment.ts (hunk 2 above)
Comment (tapabratadey) [reply src/backend/controller/comment/comment.ts]: removed

**Pull request closed without merging**